src/Symfony/Component/HttpFoundation/Tests/ParameterBagTest.php

```diff
@@ -144,6 +144,15 @@ public function testGetInt()
         $this->assertEquals(0, $bag->getInt('unknown'), '->getInt() returns zero if a parameter is not defined');
     }
 
+    public function testGetString()
+    {
+        $bag = new ParameterBag(['first' => 'shtikov', 'second' => 123]);
+
+        $this->assertEquals('shtikov', $bag->getString('first'), '->getString() gets a value of parameter as string');
+        $this->assertEquals('123', $bag->getString('second'), '->getString() gets a value of parameter as string');
```

Comment:
I think it would be better to use `assertSame` instead of `assertEquals` here as it also verifies that the type matches.
(This `assertEquals` would pass even if it the actual return value was int `123`).

Funny fact: I tried to find some issue/PR in Symfony repo where this was discussed and I found https://github.com/symfony/symfony/pull/35793 which links to my issue in the PHPUnit docs from four years ago :-) https://github.com/sebastianbergmann/phpunit-documentation-english/issues/3

Reply:
Even if the type check was not necessary, that's a good practice to use `assertSame` by default.

Reply:
Done